core/server/api/v2/utils/serializers/output/db.js

```diff
@@ -0,0 +1,41 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:db');
+
+module.exports = {
+    backupContent(filename, apiConfig, frame) {
+        debug('backupContent');
+
+        frame.response = {
+            db: [{filename: filename}]
+        };
+    },
+
+    exportContent([exportedData, fileName], apiConfig, frame) {
+        debug('exportContent');
+
+        frame.response = {
+            filename: fileName,
```

Comment:
Need to rethink how to pass filename differently, so it's not included in the response object but still available for headers stage to be used :thinking: